ref-exchange/src/account_deposit.rs

```diff
@@ -43,13 +43,20 @@ impl Account {
 
     /// Returns amount of $NEAR necessary to cover storage used by this data structure.
     pub fn storage_usage(&self) -> Balance {
-        (MIN_ACCOUNT_DEPOSIT_LENGTH + self.tokens.len() as u128 * (MAX_ACCOUNT_LENGTH + 16))
+        // [AUDIT_01]
+        (MIN_ACCOUNT_DEPOSIT_LENGTH + self.tokens.len() as u128 * (4 + MAX_ACCOUNT_LENGTH + 16))
```

Comment:
Hm, let's add 4 to MAX_ACCOUNT_LENGTH and rename to `MAX_ACCOUNT_BYTES`. 
Because it's used in other places. Like in `MIN_ACCOUNT_DEPOSIT_LENGTH` which also should be `MAX_ACCOUNT_BALANCES_BYTES`.

Reply:
It's a little tricky that  in `MIN_ACCOUNT_DEPOSIT_LENGTH` the `MAX_ACCOUNT_LENGTH`  is supposed to be the key length of a LookupMap item, which has an extra prefix compare to HashMap item in the Account Structure. It's more like a `4 bytes + len(prefix)`, but I am really uncertain how to code that.